Content.Client/Atmos/Components/GasAnalyzerComponent.cs

```diff
@@ -1,77 +1,14 @@
-using Content.Client.Items.Components;
-using Content.Client.Message;
-using Content.Client.Stylesheets;
 using Content.Shared.Atmos.Components;
-using Robust.Client.UserInterface;
-using Robust.Client.UserInterface.Controls;
-using Robust.Shared.GameObjects;
-using Robust.Shared.Localization;
-using Robust.Shared.Timing;
-using Robust.Shared.ViewVariables;
 
 namespace Content.Client.Atmos.Components
 {
     [RegisterComponent]
-    internal sealed class GasAnalyzerComponent : SharedGasAnalyzerComponent, IItemStatus
+    public sealed class GasAnalyzerComponent : SharedGasAnalyzerComponent
     {
-        [ViewVariables(VVAccess.ReadWrite)] private bool _uiUpdateNeeded;
-        [ViewVariables] private GasAnalyzerDanger Danger { get; set; }
+        [ViewVariables(VVAccess.ReadWrite)]
+        public bool UiUpdateNeeded;
```

Comment:
i would hold off on touching gas analyzer because of #10976 which just removes the iitemstatus entirely

Reply:
Removed all changes of gas analyzer. Though personally I would prefer to keep old item status.